src/topology/topological_fiber_bundle.lean

```diff
@@ -979,3 +1051,111 @@ begin
 end
 
 end topological_fiber_bundle_core
+
+section topological_fiber_prebundle
+
+variables (F) {Z : Type*} [topological_space B] [topological_space F] {proj : Z → B}
+
+/-- This structure permits to define a fiber bundle when trivializations are given as local
+equivalences but there is not yet a topology on the total space. The total space is hence given a
+topology in such a way that there is a fiber bundle structure for which the local equivalences
+are also local homeomorphism and hence local trivializations. -/
+@[nolint has_inhabited_instance]
+structure topological_fiber_prebundle (proj : Z → B) :=
+(trivialization_at : B → prebundle_trivialization F proj)
+(trivialization_at_mem : ∀ x : B, x ∈ (trivialization_at x).base_set)
+(cont_triv_inter : ∀ x y : B, continuous_on ((trivialization_at x) ∘
```

Comment:
`cont_triv_inter` is really hard to understand. Since it's not something that will be typed a lot, why not use a longer but more evocative name?

Reply:
`cont_triv_on_inter`? I believe the abbreviations I am using are standard in mathlib!

Reply:
What about `continuous_triv_change`?